src/TuxMutableRenderMixin.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+var invariant = require('./TuxInvariant');
+
+// findPath FUNCTION: recursive function to discover the first path to the provided @param mutableTrait from deepSearch function
+// @param mutableTrait STRING trait to find the path of keys to within the searchObject
+// @param searchObject OBJECT:
+  // expected keys ANY: this object can contain any keys, but is expected to contain the key specified in mutableTrait
+// @param currentPath ARRAY: array of strings that lead to the specified mutableTrait within searchObject
+// @param mutablePath ARRAY (CLOSURE from deepSearch): array that will contain the first viable path in the searchObject
+var findPath = function (mutableTrait, searchObject, currentPath) {
+  // if the searchObject is undefined or null, return as the path isn't down this branch
+  if (searchObject === undefined || searchObject === null) {
+    return;
+  } else if (searchObject.hasOwnProperty(mutableTrait)) {
+    // Add mutableTrait to the end of the currentPath and set it to mutablePath and return
+    mutablePath = currentPath.concat(mutableTrait);
```

Comment:
Need to pass in `mutablePath` to `findPath`.  `findPath` will not have access to the `mutablePath` array defined within `deepSearch` when it is declared outside of the `deepSearch` function.